hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -72,7 +75,10 @@ object HealthTransformations {
             }
           )
         } else if (healthCondition.isOther) {
-          base.copy(hsConditionOtherDescription = rawRecord.getOptional(s"${prefix}_spec"))
+          val descriptionFieldName = healthCondition.descriptionSuffixOverride
+            .map(suffix => s"${dxKey.dataPrefix}_$suffix")
+            .getOrElse(s"${prefix}_spec")
+          base.copy(hsConditionOtherDescription = rawRecord.getOptional(descriptionFieldName))
```

Comment:
This is the same change as above. If anyone has thoughts on how to reduce the duplicate code, let me know.

Reply:
With just two instances it feels acceptable to have the duplicate code